ovirt-ansible-collection.spec.in

```diff
@@ -25,7 +25,6 @@ Obsoletes:	ovirt-ansible-engine-setup
 Obsoletes:	ovirt-ansible-hosted-engine-setup
 Obsoletes:	ovirt-ansible-image-template
 Obsoletes:	ovirt-ansible-infra
-Obsoletes:	ovirt-ansible-manageiq
```

Comment:
Hmm I'd leave it here in case anyone upgrade from very ancient setup to the latest ovirt-ansible-collection